src/hardware/openbench-logic-sniffer/api.c

```diff
@@ -210,15 +210,15 @@ static int config_get(uint32_t key, GVariant **data,
 		*data = g_variant_new_uint64(devc->limit_samples);
 		break;
 	case SR_CONF_PATTERN_MODE:
-		if (devc->flag_reg & FLAG_EXTERNAL_TEST_MODE)
+		if (devc->capture_flags & CAPTURE_FLAG_EXTERNAL_TEST_MODE)
```

Comment:
I first wondered about it, but since "device_flags" are added in a later patch, this makes a lot of sense. Maybe this fact could be added to the commit message,

Reply:
Hehe, that would have indeed made it more readable. Mentioned.